web3-dev/smart-contracts/deploy-from-local-node.rst

```diff
@@ -0,0 +1,86 @@
+.. index:: smart contracts, assemblyscript
+
+.. _deploy-from-local-node:
+
+Deploy a smart contract on a local node
+========================================
+
+In this section we will execute a smart contract by running your own node.
+
+To execute the smart contract you will need:
+
+- A client configured with an address having coins.
+- A smart contract compiled in WebAssembly. We will use the `hello world` contract from the :ref:`getting-started <sc-getting-started>` step.
```

Comment:
Which hello world smart contract ? I don't think any of them would execute anything. The first one has no main function, the second one requires the first one.